modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -605,8 +605,8 @@ public Vector3 XformInv(Vector3 v)
         /// See <see cref="GetEuler()"/> if you need Euler angles, but keep in
         /// mind that quaternions should generally be preferred to Euler angles.
         /// </summary>
-        /// <returns>A <see cref="Godot.Quaternion"/> representing the basis's rotation.</returns>
-        public Quaternion Quaternion()
+        /// <returns>A <see cref="Quaternion"/> representing the basis's rotation.</returns>
+        public Quaternion GetQuaternion()
```

Comment:
Can we alphabetize the method (put it next to the other Get ones)? I'm aware that it wasn't alphabetized before, but we can improve it here.

Reply:
Done. I moved it below `GetEuler()`, I also moved `GetRotationQuaternion()`.